### PR TITLE
[Azure LoadTesting] Added tests, updated readme

### PR DESCRIPTION
**Pull request by venkatr21**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/loadtestservice/azure-mgmt-loadtesting/README.md [hunk 1]
```diff
@@ -45,6 +45,75 @@ client = LoadTestMgmtClient(credential=DefaultAzureCredential(), subscription_id
 
 ## Examples
 
+### Create a new Azure Load Testing resource
+```python
+from azure.mgmt.loadtesting.models import (
```
Comment (msyyc) [inline sdk/loadtestservice/azure-mgmt-loadtesting/README.md:50]: @venkatr21 README.md of mgmt package is rendered with template automatically so your change will be overrided. You can add the samples in https://github.com/Azure-Samples/azure-samples-python-management/tree/main/samples/loadtestservice and users can find them in README.md

Comment (venkatr21) [reply sdk/loadtestservice/azure-mgmt-loadtesting/README.md:50]: @msyyc Sure, I have raised the samples here: https://github.com/Azure-Samples/azure-samples-python-management/pull/120
Should I undo the readme changes here?

Comment (msyyc) [reply sdk/loadtestservice/azure-mgmt-loadtesting/README.md:50]: @venkatr21 I merged https://github.com/Azure-Samples/azure-samples-python-management/pull/120 and please revert your change about README.md

Comment (venkatr21) [reply sdk/loadtestservice/azure-mgmt-loadtesting/README.md]: @msyyc I have reverted the readme changes which I have made

File: sdk/loadtestservice/azure-mgmt-loadtesting/tests/test_resourceoperations.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        assert loadtest_resource_patch_response.encryption.identity != None
+        assert loadtest_resource_patch_response.encryption.identity.type == "UserAssigned"
+        assert loadtest_resource_patch_response.encryption.identity.resource_id == self.msi1.id
+
+        # Delete the load test resource - returns a poller
+        loadtest_resource_delete_poller = self.loadtestservice_client.load_tests.begin_delete(resource_group.name, resource_group.name+"resource")
+
+        # Wait for the deletion LRO to complete
+        while loadtest_resource_delete_poller.done()==False:
+            loadtest_resource_delete_poller.wait(2)
```
Comment (msyyc) [inline sdk/loadtestservice/azure-mgmt-loadtesting/tests/test_resourceoperations.py:156]:
```suggestion
        self.loadtestservice_client.load_tests.begin_delete(resource_group.name, resource_group.name+"resource").result()
```

same for similar operation. In mgmt, `.result()` means that operation will wait for polling over

File: sdk/loadtestservice/azure-mgmt-loadtesting/tests/test_resourceoperations.py [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+        
+        # Wait for the poller to complete
+        while loadtest_resource_poller.done()==False:
+            loadtest_resource_poller.wait(2)
+        
+        # Get the result of the poller
+        loadtest_resource_patch_response = loadtest_resource_patch_poller.result()
+
+        # Assert the result of the poller
+        assert loadtest_resource_patch_response != None
```
Comment (msyyc) [inline sdk/loadtestservice/azure-mgmt-loadtesting/tests/test_resourceoperations.py:138]:
```suggestion
        assert loadtest_resource_patch_response
```
same for other lines

File: sdk/loadtestservice/azure-mgmt-loadtesting/tests/mgmt_test_helper.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+                            'keys': ['all']
+                        }
+                    }
+                ]
+            }
+        }
+    )
+    while resource_poller.done()==False:
+        resource_poller.wait(2)
+    akv = resource_poller.result()
```
Comment (msyyc) [inline sdk/loadtestservice/azure-mgmt-loadtesting/tests/mgmt_test_helper.py:48]:
```suggestion
    akv = resource_poller.result()
```

File: sdk/loadtestservice/azure-mgmt-loadtesting/tests/test_resourceoperations.py [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+        loadtest_resource_delete_poller = self.loadtestservice_client.load_tests.begin_delete(resource_group.name, resource_group.name+"resource")
+
+        # Wait for the deletion LRO to complete
+        while loadtest_resource_delete_poller.done()==False:
+            loadtest_resource_delete_poller.wait(2)
+
+
+#------------------------------------------------------------------------------
+if __name__ == '__main__':
+    unittest.main()
```
Comment (msyyc) [inline sdk/loadtestservice/azure-mgmt-loadtesting/tests/test_resourceoperations.py:161]: please use `black` to format your test code:
```
pip install black
black tests/
```

Comment (venkatr21) [reply sdk/loadtestservice/azure-mgmt-loadtesting/tests/test_resourceoperations.py]: updated formatting by using black as mentioned

File: sdk/loadtestservice/azure-mgmt-loadtesting/tests/mgmt_test_helper.py (hunk 4 above)
Comment (venkatr21) [reply sdk/loadtestservice/azure-mgmt-loadtesting/tests/mgmt_test_helper.py]: Updated tests.

File: sdk/loadtestservice/azure-mgmt-loadtesting/tests/test_resourceoperations.py (hunk 3 above)
Comment (venkatr21) [reply sdk/loadtestservice/azure-mgmt-loadtesting/tests/test_resourceoperations.py]: Updated it at all required lines

**Review by iscai-msft — APPROVED** [review]

**Pull request merged**